tests/Duckling/AmountOfMoney/EN/Tests.hs

```diff
@@ -144,7 +144,7 @@ intersectTests = testCase "Intersect Test" $
          , ("1.1 dollars 6", 2)
          , ("10 cents and 8 cents", 2)
          , ("10 dollars and 2 dollars", 2)
-         , ("between 4.1 dollars and 4 dollars", 3)
+         , ("between 4.1 dollars and 4 dollars", 2)
```

Comment:
Does this means the body we classified from "between 4.1 dollars and 4 dollars" was "4.1 dollars", "dollars and 4", and "4 dollars"; and it becomes "4.1 dollars" and "4 dollars" after we change the rules? Just not sure how the 3 and 2 comes from.

Reply:
I think that's accurate. I believe that the test case is tracking the number of unique parses for the input where token ranges which are subsets of another token range are excluded:

- "4.1 dollars"
- "4 dollars"
- "1 dollars and 4"

My change removed the rule which allowed for the "1 dollars and 4" parse.

The classifier seems to only be taken into consideration for figuring out the prioritization for applying rules for identical token ranges:

- ((after march) 2019)
- (after (march 2019))